config/env_blank.json

```diff
@@ -26,5 +26,9 @@
     "dbname": "",
     "user": "",
     "password": ""
-  }
+  },
+  "ZOOM_TOKEN": "<ADD THIS TOKEN>",
+  "ZOOM_BASE_URL": "<ADD zoom.us URL>",
+  "ZOOM_EARLIEST_FROM": "2020-03-01"
```

Comment:
Can you add these to the table in `README.md`?

Reply:
Thanks I'll add these, this is why I wanted the YAML ;) I'll add to the YAML issue to move the readme comments into the YAML.

Reply:
Maybe I’m just being protective of my work, but it seems like the table doesn’t need to be removed. Extensive in-line comments can get messy; I’d prefer to have an external key. Also, you can’t as easily link or format comments in YAML.